### PR TITLE
Change required description on Safari

### LINKED ISSUE #447: Safari has already supported the "required" attribute
#### 浏览器信息
* 浏览器名称，版本号
Safari Version 11.0.2 (13604.4.7.1.3)
* 操作系统
macOS High Sierra
#### 问题描述
Use HTML5 to Require a Field
Safari has already supported the "required" attribute
* 简要说明如何重现
Use Safari to complete this lecture

------

Add by S1ngS1ng
1. [Code reference](https://github.com/FreeCodeCampChina/freecodecamp.cn/blob/dev/seed/challenges/01-front-end-development-certification/html5-and-css.json#L1340)
2. [caniuse.com reference](https://caniuse.com/#search=required), `version > 10.1` as specified

### PR DESCRIPTION
**Pull request by wudifeixue**
In this PR:

Fixes #447

### CODE REVIEW

**Review by S1ngS1ng — CHANGES_REQUESTED** [review]
Typo :P

File: seed/challenges/01-front-end-development-certification/html5-and-css.json [hunk 1]
```diff
@@ -1337,7 +1337,7 @@
         "例如，如果你想把一个文本输入字段设置为必填项，在你的<code>input</code>元素中加上<code>required</code>属性就可以了，你可以使用：<br><code>&#60;input type=\"text\" required&#62;</code>",
         "任务：给你的输入框添加<code>required</code>属性，这样用户不填写输入框就无法提交表单。",
         "试试输入框不填写任何文本，然后点击Submit提交表单，看看浏览器如何提示你该字段为必填项。",
-        "注意：<code>required</code>属性在Safari浏览器中不起作用，请用其他浏览器来学习，推荐使用Chrome。"
+        "注意：<code>required</code>属性在Safari浏览器10.1以下不起作用（2017年3月27以前)，请更更新Safari或者使用其他浏览器来学习，推荐使用Chrome。"
```
Comment (S1ngS1ng) [inline seed/challenges/01-front-end-development-certification/html5-and-css.json]: "请更更新" => "请更新"

**Review by S1ngS1ng — CHANGES_REQUESTED** [review]

File: seed/challenges/01-front-end-development-certification/html5-and-css.json (hunk 1 above)
Comment (S1ngS1ng) [reply seed/challenges/01-front-end-development-certification/html5-and-css.json]: Also, please unify the symbols (parenthesis here) to EITHER full-width OR half-width. Mix & match won't be a good idea 😂

Comment (wudifeixue) [reply seed/challenges/01-front-end-development-certification/html5-and-css.json]: 哈哈好尴尬呀居然打错了字，那我括号也改一改好了

**Comment by wudifeixue** [comment]
Updated now

**Review by S1ngS1ng — APPROVED** [review]

**Pull request merged**